pontoon/teams/views.py

```diff
@@ -173,15 +173,14 @@ def ajax_permissions(request, locale):
 
 @login_required(redirect_field_name='', login_url='/403')
 @require_POST
-def request_item(request, locale=None):
+def request_item(request, type=None, code=None):
```

Comment:
`type` is [not a good](https://stackoverflow.com/questions/2417979/can-i-use-the-variable-name-type-as-function-argument-in-python) variable name in Python.